### PR TITLE
Fix(curriculum): external Tower of Hanoi link in CIP

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Tower of hanoi #40749

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md [hunk 1]
```diff
@@ -8,13 +8,13 @@ dashedName: towers-of-hanoi
 
 # --description--
 
-Solve the [Towers of Hanoi](https://en.wikipedia.org/wiki/Towers_of_Hanoi "wp: Towers_of_Hanoi") problem.
+Solve the Towers of Hanoi problem. You have the number of objects as the first parameters, and strings used to identify each of the stacks of objects. create a nested array containing the list of moves, source -> destination.
+ 
+For example, object with these parameters `(4, 'A', 'B', 'C')`, will have a nested array `[['A', 'C'], ['B', 'A']]` indicating that the 1st move was to move a object from stack `A` to `C`, and the 2nd move was to move a object from stack `B` to `A`.
```
Comment (Sembauke) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md:13]:
```suggestion
For example, objects with these parameters `(4, 'A', 'B', 'C')`, will have a nested array `[['A', 'C'], ['B', 'A']]` indicating that the 1st move was to move an object from stack `A` to `C`, and the 2nd move was to move an object from stack `B` to `A`.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md [hunk 2]
```diff
@@ -8,13 +8,13 @@ dashedName: towers-of-hanoi
...
 
-Your solution should accept the number of discs as the first parameters, and three string used to identify each of the three stacks of discs, for example `towerOfHanoi(4, 'A', 'B', 'C')`. The function should return an array of arrays containing the list of moves, source -> destination.
+# --instructions--
 
-For example, the array `[['A', 'C'], ['B', 'A']]` indicates that the 1st move was to move a disc from stack A to C, and the 2nd move was to move a disc from stack B to A.
-
-<p></p>
+Write a function that return the moves to stack the objects in a nested array.
```
Comment (Sembauke) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md:17]:
```suggestion
Write a function that returns the moves to stack the objects in a nested array.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — DISMISSED** [review]

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md [hunk 3]
```diff
@@ -8,13 +8,13 @@ dashedName: towers-of-hanoi
 
 # --description--
 
-Solve the [Towers of Hanoi](https://en.wikipedia.org/wiki/Towers_of_Hanoi "wp: Towers_of_Hanoi") problem.
+Solve the Towers of Hanoi problem. You have the number of objects as the first parameters, and strings used to identify each of the stacks of objects. create a nested array containing the list of moves, source -> destination.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md:11]: perhaps:
```suggestion
Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, and followed by the strings used to identify each of the stacks of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md [hunk 4]
```diff
@@ -8,13 +8,13 @@ dashedName: towers-of-hanoi
...
+ 
+For example, objects with these parameters `(4, 'A', 'B', 'C')`, will have a nested array `[['A', 'C'], ['B', 'A']]` indicating that the 1st move was to move an object from stack `A` to `C`, and the 2nd move was to move an object from stack `B` to `A`.
```
Comment (jeremylt) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md:13]:
```suggestion
For example, the parameters `(4, 'A', 'B', 'C')`, will result in nested array of moves `[['A', 'C'], ['B', 'A']]`, indicating that the 1st move was to move an object from stack `A` to `C` and the 2nd move was to move an object from stack `B` to `A`.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md [hunk 5]
```diff
@@ -8,13 +8,13 @@ dashedName: towers-of-hanoi
 
 # --description--
 
-Solve the [Towers of Hanoi](https://en.wikipedia.org/wiki/Towers_of_Hanoi "wp: Towers_of_Hanoi") problem.
+Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, and followed by the strings used to identify each of the stacks of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```
Comment (Nirajn2311) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md:11]:
```suggestion
Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, followed by the strings used to identify each stack of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by Nirajn2311 — APPROVED** [review]
LGTM 👍

**Review by raisedadead — APPROVED** [review]

**Pull request merged**